src/flow/core/data.rs

```diff
@@ -225,6 +225,26 @@ impl<'a> From<&'a str> for DataType {
     }
 }
 
+impl fmt::Debug for DataType {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match *self {
+            DataType::None => write!(f, "None"),
+            DataType::Text(..) => {
+                let text: Cow<str> = self.into();
+                write!(f, "Text ({:?})", text)
```

Comment:
Can we remove the extra space before the `(` to stay consistent with normal Rust debug output?

Reply:
Fixed in https://github.com/mit-pdos/distributary/pull/25/commits/f031b608e1377165d0e20d5ac7181557ff65aa99